### PR TITLE
Najla practice

### PR DESCRIPTION
**Pull request by NajlaSoltana**

### CODE REVIEW

File: week-1-practise/level-1/1-syntax-error-1.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
+function addNumbers(a ,b ,c) {
```
Comment (foaly-nr1) [inline week-1-practise/level-1/1-syntax-error-1.js]: Just a minor detail, but please make sure to have a space after `,` and no space before. Consistency is important.

File: week-1-practise/level-2/4-currency-conversion-2.js [hunk 2]
```diff
@@ -7,8 +7,18 @@
...
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var result = price * 1.4;
+  var convertFee = result * 0.01;
+  result += convertFee;
+  return result.toFixed(2);
+}
+function convertToBRL(price) {
+  var result = price * 4.93;
+  var convertFee = result * 0.01;
```
Comment (foaly-nr1) [inline week-1-practise/level-2/4-currency-conversion-2.js]: Can you think of a way to apply the DRY principle (don't repeat yourself) here? This is a very good example of code that can be improved through DRY – you have 2 identical lines of code!

File: week-1/I-floats/exercise.js [hunk 3]
```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+var perstudent = Math.round((100 * numberOfStudents) / total);
+var permentors = Math.round((100 * numberOfMentors) / total);
```
Comment (foaly-nr1) [inline week-1/I-floats/exercise.js]: Can you think of a way to use a function to make this more DRY (don't repeat yourself)?

File: week-1/J-functions/exercise.js [hunk 4]
```diff
@@ -1,7 +1,5 @@
 function halve(number) {
-  // complete the function here
+  return number;
```
Comment (foaly-nr1) [inline week-1/J-functions/exercise.js]: Double check the function name. Do the name and behaviour match?

File: week-1/L-functions-nested/exercise.js [hunk 5]
```diff
@@ -1,5 +1,13 @@
...
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+
+var students = 104;
+var mentors = 56;
+var total = students + mentors;
+var percentageOfMentors = Math.round((mentors*100)/total);
+var percentageOfStudents = Math.round((students*100)/total);
```
Comment (foaly-nr1) [inline week-1/L-functions-nested/exercise.js]: If you move these 5 variables below the function description, does it still work? Can you think of a way to use function parameters?

If you want some additional reading on parameters, this one might help: https://codeburst.io/parameters-arguments-in-javascript-eb1d8bd0ef04

File: week-1/L-functions-nested/exercise2.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+
+function mentors ( names) {
+    return names.toUpperCase(names);
```
Comment (foaly-nr1) [inline week-1/L-functions-nested/exercise2.js]: There's a little bug here. Once you've googled the method definition of `toUpperCase` can you spot it?

File: week-1/L-functions-nested/exercise2.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+}
+function greetings (names){
+    return "HELLO "+ mentors(names)
```
Comment (foaly-nr1) [inline week-1/L-functions-nested/exercise2.js]: I was a little confused by your choice of variable name. How many people can I greet at once using this function?

File: week-1-practise/level-1/10-function-output-3.js [hunk 8]
```diff
@@ -1,5 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
+    return firstWord.concat(secondWord, thirdWord);
 }
 
 var result = concatenate('code', 'your', 'future');
```
Comment (clems27) [inline week-1-practise/level-1/10-function-output-3.js]: Nice work, I think you did a great job with your concentration, welldone

File: week-1-practise/level-2/2-format-currency.js [hunk 9]
```diff
@@ -6,7 +6,10 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(currency) {
+  var result = "£" + currency.toFixed(2);
+  return result
```
Comment (clems27) [inline week-1-practise/level-2/2-format-currency.js]: Great way to use the .toFixed() method, I see you are enjoying coding, well done

**Pull request closed without merging**